### PR TITLE
Debt/llcbi 11068

### PR DESCRIPTION
**Pull request by joshkraszeski**
Adding header and p tag styles to LDS
Build: https://ncinodev.atlassian.net/builds/browse/MANAGEDAPPS-DSCI11-3

DO NOT MERGE YET!
Reviewed by @richykandora 
Reviewed by @matt-lukens

### CODE REVIEW

File: ui/core/core-bs3.scss [hunk 1]
```diff
@@ -18,6 +18,85 @@
...
+		padding-top: $spacing-small;
+		padding-right: $spacing-medium;
+		padding-left: $spacing-medium;
+		padding-bottom: $spacing-small;
+		text-transform: uppercase;
+	}
+
+	p{
+		font-family: $font-family;
+		color: $color-text-default;
```
Comment (richykandora) [inline ui/core/core-bs3.scss:91]: alphabetize

**Comment by richykandora** [comment]
looks good!

File: ui/core/core-bs3.scss [hunk 2]
```diff
@@ -18,6 +18,85 @@
...
 
+  h1{
+		font-family: $font-family;
+		font-weight: $font-weight-light;
+		font-size: $font-size-xx-large;
+		line-height: $line-height-heading;
+		margin-top: $spacing-large;
+		margin-right: $spacing-large;
+		margin-bottom: $spacing-none;
+		margin-left: $spacing-large;
```
Comment (matt-lukens) [inline ui/core/core-bs3.scss:29]: These repeated directives are a great opportunity to define and use a mixin.  You could define a heading mixin that includes the margin, line-height, font-family and font-weight directives, then include it in each tag.

File: ui/core/core-bs3.scss [hunk 3]
```diff
@@ -18,6 +18,85 @@
...
+	}
+
+
+	h6{
+		background: $color-background;
+		color: $color-text-weak;
+		font-family: $font-family;
+		font-size: $font-size-small;
+		letter-spacing: .0625em;
```
Comment (matt-lukens) [inline ui/core/core-bs3.scss:81]: Needs to be a token please.

**Pull request closed without merging**